examples/nodejs/aws/eventbridge/README.template.md

```diff
@@ -0,0 +1,52 @@
+# Welcome to Momento <-> Eventbridge Integration Example Project
```

Comment:
you need to update the github workflow files for PR and push, to tell them about this template file that needs to be generated.